static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js

```diff
@@ -1,152 +0,0 @@
-// @flow
-
-import { acquisitionsBannerFivTemplate } from 'common/modules/commercial/templates/acquisitions-banner-fiv';
```

Comment:
personally I'd argue for removing `acquisitions-banner-fiv` and re-instating later if we need it, since it will affect the JS bundle  size and it's not clear that the design will be re-used as is

Reply:
👍 will delete it